tests/regression/test_fdm.py

```diff
@@ -255,3 +295,48 @@ def test_direct_solver(fs):
 
     assert solver.snes.ksp.getIterationNumber() == 1
     assert norm(u_exact-uh, "H1") < 1.0E-8
+
+
+@pytest.mark.skipcomplex
+def test_static_condensation(mesh):
```

Comment:
This doesn't seem related to the main thrust of this PR?

Reply:
FDMPC automatically statically condenses the problem if given a facet element, this is just testing SLATE's matfree interior solvers and using FDMPC on the Schur complement.

Reply:
There is another similar test that I might need to include, because FDMPC is supposed to handle MixedFunctionSpaces that arise from an interior-facet decomposition.